Week2/homework/map_filter.js

```diff
@@ -3,3 +3,7 @@
 const numbers = [1, 2, 3, 4];
 
 // Add your code here
+const newNumbers = numbers.filter(x => x %2 ).map(x => x * 2);
+
+console.log(`the double numbers are`, newNumbers);
```

Comment:
You should avoid using backticks in this console log since you are not using any placeholders. Use normal quotes/double-quotes instead.